src/core/environment.js

```diff
@@ -246,12 +246,18 @@ p5.prototype.cursor = function(type, x, y) {
  * frames per second (usual for movies) or above will be enough for smooth
  * animations. This is the same as setFrameRate(val).
  *
- * Calling <a href="#/p5/frameRate">frameRate()</a> with no arguments returns
- * the current framerate. The draw function must run at least once before it will
- * return a value. This is the same as <a href="#/p5/getFrameRate">getFrameRate()</a>.
- *
- * Calling <a href="#/p5/frameRate">frameRate()</a> with arguments that are not
- * of the type Number or are non-positive also returns current framerate.
+ * Calling <a href="#/p5/frameRate">frameRate()</a> with no arguments or
+ * with arguments that are not of type Number or are non-positive returns
+ * an approximation of the current frame rate. The draw function must run at
+ * least once before it will return a value. This is the same as using
+ * <a href="#/p5/getFrameRate">getFrameRate()</a>.
+ *
+ * The returned frame rate value is technically the rate at which the draw 
+ * function is called per second. This value can vary wildly, even if the 
+ * actual display rate achieved is consistent. To accurately test the
```

Comment:
It might be a bit confusing to readers describing the display rate achieved, as the display rate of the monitor is likely fixed and not related to the sketch, and the draw rate of the sketch itself is what `frameRate()` returns. Do you mean that the speed at which the code in the `draw` function executes can be constant? How would you feel about saying something like this: "Even if the code in `draw()` takes a consistent amount of time to run, the value of `frameRate()` can vary from frame to frame as p5 attempts to achieve a target frame rate on average."